src/duckling/forms.scss

```diff
@@ -6,12 +6,18 @@ $form-label-padding: $form-label-padding-top $form-label-padding-right $form-lab
 
 $spacer-unit: 13px;
 
-.form-label {
+.form-label:not(:first-child) {
```

Comment:
You could dry this up by nesting the first-child selector within form-label and overriding the padding there.